index.bs

```diff
@@ -1058,6 +1085,18 @@ that determine how WebTransport connection is established and used.
    use for normal [=fetch=] operations.
 :: This cannot be used with {{WebTransportOptions/allowPooling}}.
 
+: <dfn for="WebTransportOptions" dict-member>congestionControl</dfn>
+:: When set, influences the congestion control algorithm used by the user agent
+   when sending data over this connection.
+   <div class="issue atrisk">
+     <p>
+       This configuration option is considered at risk due to the lack of
+       implementation and specification of any congestion control algorithm
+       other than [[!RFC9002]] [section 7](https://datatracker.ietf.org/doc/html/rfc9002#section-7)
+       at the time of writing.
```

Comment:
What Lucas said.  The challenge here is in standardizing an algorithm that optimizes for latency.  So maybe say that instead.